signac/core/collection_mongodb.py

```diff
@@ -40,7 +40,7 @@ def _sync(self):
         except self._errors.InvalidDocument as err:
             raise TypeError(str(err))
 
-    def __deepcopy__(self, memo):
+    def _pseudo_deepcopy(self, memo=None):
```

Comment:
- [ ] Please add a doc-string that describes the purpose of this function and the motivation for its implementation.
- [ ] Please remove the memo argument, because its use should be restricted to the `__deepcopy__` signature.